### ISSUE TITLE
Mailer Functions

### ISSUE THREAD

**Issue by Michaeldgeek** [Michaeldgeek/NacossUnn#1]
The default mail function in php actually works but gmail recognizes it as a spam mail.
I need a mailer function that can deliver mail and will not be treated as spam.

**Comment by AVONnadozie** [comment]
A mail being treated as spam has nothing to do with the mail function sending it but the mail itself.
Mails are marked as spam mainly by its headers and content, the best thing to do is to checkout guidelines such as http://wiki.apache.org/spamassassin/AvoidingFpsForSenders and investigate where your mail has gone wrong.
A getaway might be to send the mails in plain text by setting content-type to text/plain and setting proper header information (the current one looks ok).

In a nutshell, check the link and know what you have to add or remove to make the mail look less like a spam.
